ultrasync/main.py

```diff
@@ -599,8 +599,22 @@ def set_zone_bypass(self, zone, state=False):
                 'opt': int(state),
                 'zone': zone - 1,
             })
-
-        else:  # self.vendor is NX595EVendor.{COMNAV, ZEROWIRE, XGEN}
+        elif self.vendor in (NX595EVendor.COMNAV):
+            # Call comnav_process_zones to update can_bypass attribute
+            self.comnav_process_zones
+
+            # Get the current can_bypass state of the zone
+            can_bypass = self.zones[zone - 1]['can_bypass']
+            # If the current can_bypass state does not match the desired bypass
+            # state, toggle the bypass state
+            if can_bypass == state:
+                # Start our payload off with our session identifier
+                payload = {
+                    'sess': self.session_id,
+                    'comm': 82,
+                    'data0': zone - 1,
+                }
+        else:
```

Comment:
Just for clarity can you add:
```python
else   # self.vendor is NX595EVendor.{ZEROWIRE, XGEN}
```

I realize you accommodate COMNAV now, but in the future (should we look at this code in say 1-2 years from now, it will still make sense).

Otherwise, it seems you pretty much captured the other elements.

Reply:
Sure. Makes sense. The latest commit adds that line.